overlay/root/Makefile.sample

```diff
@@ -0,0 +1,20 @@
+NAME =                  APP_NAME
```

Comment:
it 's difficul to understand the difference between NAME, TITLE and IMAGE_NAME, so let's use possible values

here: my-image